packages/chart/src/components/EditorPanel/EditorPanel.tsx

```diff
@@ -1897,6 +1894,31 @@ const EditorPanel: React.FC<ChartEditorPanelProps> = ({ datasets }) => {
                       </Tooltip>
                     }
                   />
+                  <CheckBox
+                    display={!visHasCategoricalAxis() && config.visualizationType === 'Pie'}
```

Comment:
Can this be simplified to just config.visualizationType === Pie?

Reply:
Fixed